packages/tools/devtools/devtools-browser-extension/src/BackgroundConnection.ts

```diff
@@ -140,8 +141,12 @@ export class BackgroundConnection
 
 		// Ignore messages from unexpected sources.
 		// We receive at least one message directly from the Background script so we need to include
-		// extensionMessageSource as a valid source.
-		if (message.source !== extensionMessageSource && message.source !== devtoolsMessageSource) {
+		// extensionMessageSource and extensionPopupMessageSource as valid sources.
+		if (
+			message.source !== extensionMessageSource &&
```

Comment:
Nit: Can we rename this existing message source to `extensionViewMessageSource` and update its docs to be specific to the extension view? Just to help differentiate it a bit better from our new source.